Apps/Common/Test/LR/TestMultiPatchLRRefine.C

```diff
@@ -19,10 +19,12 @@
 #include "SIM3D.h"
 #include "tinyxml2.h"
 
-#include "gtest/gtest.h"
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/generators/catch_generators.hpp>
 
 #include <sstream>
 
+namespace {
```

Comment:
Don't quite see the point of putting these classes in anonymous namespace in the Test C-file. Is it because they are also defined in other Test-files (but with different content) compiled into the same binary? Or some other reason.